cmd/main.go

```diff
@@ -13,12 +13,14 @@ import (
 
 var (
 	// Common parameters for all commands
-	debug         bool
-	organization  string
-	neo4jURI      string
-	neo4jUser     string
-	neo4jPassword string
-	session       string
+	debug            bool
+	organization     string
+	neo4jURI         string
+	neo4jUser        string
+	neo4jPassword    string
+	session          string
+	githubApiURI     string
```

Comment:
Since these parameters are for the GitHub command, could we shift them below? next to `githubToken`?

Reply:
fixed